client/settings/check-starcraft-path.js

```diff
@@ -64,7 +74,10 @@ export async function checkStarcraftPath(dirPath, downgradePath) {
     checkHash(path.join(downgradePath, 'storm.dll'), STORM_HASHES_1161),
   ])
 
-  if (starcraftValid && stormValid) {
+  matches = await globAsync(`${downgradePath}/local.dll`)
```

Comment:
Why are you globbing this, exactly? There's no pattern here whatsoever, just access it like we do above